test/PowerShellEditorServices.Test/Session/PsesInternalHostTests.cs

```diff
@@ -170,10 +170,29 @@ public async Task CanHandleNoProfiles()
         [Fact]
         public async Task CanHandleBrokenPrompt()
         {
-            await psesHost.ExecutePSCommandAsync(
-                new PSCommand().AddScript("function prompt { throw }"),
+            _ = await Assert.ThrowsAsync<RuntimeException>(() =>
+            {
+                return psesHost.ExecutePSCommandAsync(
+                    new PSCommand().AddScript("function prompt { throw }; prompt"),
+                    CancellationToken.None);
+            }).ConfigureAwait(true);
+
+            string prompt = await psesHost.ExecuteDelegateAsync(
+                nameof(psesHost.GetPrompt),
+                executionOptions: null,
+                (_, _) => psesHost.GetPrompt(CancellationToken.None),
                 CancellationToken.None).ConfigureAwait(true);
 
+            Assert.Equal(PsesInternalHost.DefaultPrompt, prompt);
+        }
+
+        [Fact]
+        public async Task CanHandleUndefinedPrompt()
+        {
+            Assert.Empty(await psesHost.ExecutePSCommandAsync<PSObject>(
+                new PSCommand().AddScript("Remove-Item function:prompt; Get-Item function:prompt"),
```

Comment:
```suggestion
                new PSCommand().AddScript("Remove-Item function:prompt; Get-Item function:prompt -ErrorAction Ignore"),
```

This (ideally) writes a non-terminating error. Since we don't care about it, we should set error action ignore in case we ever want to set error action for tests.